pandas/core/indexes/multi.py

```diff
@@ -2925,7 +2931,9 @@ def _partial_tup_index(self, tup: tuple, side: Literal["left", "right"] = "left"
             if lab not in lev and not isna(lab):
                 # short circuit
                 try:
-                    loc = algos.searchsorted(lev, lab, side=side)
+                    # Argument 1 to "searchsorted" has incompatible type "Index";
```

Comment:
This is surprising to see?

Reply:
Yeah, I think our typing isn't entirely correct somewhere